app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt

```diff
@@ -43,14 +44,25 @@ class AuthActivity : AppCompatActivity() {
 
         supportActionBar?.title = "Login"
 
-        loadFragment(LoginFragment())
+        checkAndLoadFragment(LoginFragment())
+    }
+
+    private fun checkAndLoadFragment(fragment: Fragment) {
```

Comment:
@iamareebjamal should we move `checkAndLoadFrag` and `loadFragment` to the utility class?

Reply:
![untitled](https://user-images.githubusercontent.com/29306481/47020400-e7cb3300-d176-11e8-8818-5240b546fcc7.png)
it is showing this!

Reply:
I'll still change it to previous version and add a commit.